### PR TITLE
Added a few more sites

### PR DESCRIPTION
**Pull request by sethazazel**
AnimeKayo and Bakabt added, AnimeKaizoku moved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +85,7 @@ A curated list of anime streaming/downloading websites.
 
  * [animebytes.tv](https://animebytes.tv/)
  * [animetorrents.me](https://animetorrents.me/)
+ * [bakabt.me](https://www.bakabt.me/)
```
Comment (anshumanv) [inline README.md:88]: What's this? Seems to be asking a username and pass

Comment (sethazazel) [reply README.md:88]: Private torrent tracker for anime. You can request invitation from irc channel. They will ask a few questions and proof for ratio from other private trackers. [https://wiki.bakabt.me/index.php/Sign_up](https://wiki.bakabt.me/index.php/Sign_up)

Comment (anshumanv) [reply README.md:88]: Ah cool! Maybe we should add this info as well as the users you are landing on the page won't have this info.

Comment (anshumanv) [reply README.md:88]: Just add a note near the link and it should be good

**Comment by anshumanv** [comment]
Hi @sethazazel Sorry for the delay, can you rebase? We can merge after that :)

**Comment by toprak** [comment]
They are already in the list, feel free to add more sites!

**Pull request closed without merging**